### PR TITLE
Remove deprecated command 'kubectl stop'

### PR DESCRIPTION
**Pull request by shiywang**
Fixes https://github.com/kubernetes/kubernetes/issues/11384
I think this is the final step @kubernetes/sig-cli-pr-reviews  @kargakis

### CODE REVIEW

**Comment by shiywang** [comment]
@k8s-bot pull-kubernetes-unit test this

File: pkg/kubectl/cmd/cmd.go [hunk 1]
```diff
@@ -355,11 +355,7 @@ func NewKubectlCommand(f cmdutil.Factory, in io.Reader, out, err io.Writer) *cob
 	}
 	groups.Add(cmds)
 
-	filters := []string{
-		"options",
-		deprecated("kubectl", "delete", cmds, NewCmdStop(f, out)),
```
Comment (kargakis) [inline pkg/kubectl/cmd/cmd.go]: Just remove this line that is relevant for your changes

Comment (shiywang) [inline pkg/kubectl/cmd/cmd.go]: oh, I'm sorry, there's "options" in that array, read it wrong

**Comment by shiywang** [comment]
@kargakis  updated, ptal

**Comment by kargakis** [comment]
> @kargakis updated, ptal

Still the same code

**Comment by shiywang** [comment]
@kargakis really? I think I already updated, but anyway I pushed again, is this time still the same code ? the comment you leave didn't folded because I removed four line code and add a new line instead.

**Comment by kargakis** [comment]
Seems fine now - do you also need to run some update scripts (bazel, generated-docs maybe)?

**Comment by shiywang** [comment]
@kargakis I think I already did, will wait for all tests goes green, and then I'll ping you : )

**Comment by kargakis** [comment]
LGTM - needs a sign-off from @kubernetes/sig-cli-api-reviews

**Comment by shiywang** [comment]
@k8s-bot pull-kubernetes-federation-e2e-gce test this

**Comment by shiywang** [comment]
I think the test failed is because merged failed and cause by that pr

**Comment by shiywang** [comment]
@kubernetes/sig-cli-api-reviews

**Comment by mengqiy** [comment]
@pwittrock Need your approval for `/docs` folder.

**Comment by shiywang** [comment]
/assign @pwittrock 
for approval

**Comment by xiangpengzhao** [comment]
xref: https://github.com/kubernetes/kubernetes/pull/49935

**Comment by shiywang** [comment]
rebased, @mengqiy mind relabel it ?

**Pull request merged**